### DISCUSSION TITLE
Proposed Request flow diagram for invoice-based-loans

### DISCUSSION THREAD

**Discussion by recur-anirudh** [ONDC-Official/ONDC-FIS-Specifications#4] [General]
As we went through the shared swagger for ONDC protocol, is there a request flow diagram that using which we can map the APIs to an action in the invoice-based-loans journey.

Also, how can one track the status of the repayments and how is pre-payment supported.

**Comment by Antriksh-Parmar** [comment]
@92shreyansh can you pls respond to this query.

To your second question, functionally, the BAP can request for payment link from the lender to support pre-payment/late-payment. Repayments are done via eMandate/eNACH/Standing instruction.

**Comment by 92shreyansh** [comment]
The Flow UI encomprise of all the API in a sequence along with the illustrative flow which can be used to map a user journey.
